Cubical/Algebra/Group/Abelianization/AbelianizationAsCoeq.agda

```diff
@@ -0,0 +1,354 @@
+
+  -- Definition of the group structure on the abelianization. Here the generality of the glued functions is used.
+  _·Ab_ : Abelianization → Abelianization → Abelianization
+  _·Ab_ =
+    rec2
+      squash
+      (λ x y → incAb (x · y))
+      (λ (a , b , c) d → incAb ((a · (b · c)) · d) ≡⟨ cong (λ x → incAb (x · d)) (assoc _ _ _) ⟩
+                         incAb (((a · b) · c) · d) ≡⟨ cong incAb (sym (assoc (a · b) c d)) ⟩
+                         incAb ((a · b) · (c · d)) ≡⟨ comm (a · b) c d ⟩
+                         incAb ((a · b) · (d · c)) ≡⟨ cong incAb (sym (assoc _ _ _)) ⟩
+                         incAb (a · (b · (d · c))) ≡⟨ cong (λ x → incAb (a · x)) (assoc _ _ _) ⟩
+                         incAb (a · ((b · d) · c)) ≡⟨ comm a (b · d) c ⟩
+                         incAb (a · (c · (b · d))) ≡⟨ cong (λ x → incAb (a · x)) (assoc _ _ _) ⟩
+                         incAb (a · ((c · b) · d)) ≡⟨ cong incAb (assoc a (c · b) d) ⟩
+                         incAb ((a · (c · b)) · d) ∎)
+      (λ (b , c , d) a → incAb (a · (b · (c · d))) ≡⟨ cong incAb (assoc _ _ _) ⟩
+                         incAb ((a · b) · (c · d)) ≡⟨ comm (a · b) c d ⟩
+                         incAb ((a · b) · (d · c)) ≡⟨ cong incAb (sym (assoc _ _ _)) ⟩
+                         incAb (a · (b · (d · c))) ∎)
+
+  1Ab : Abelianization
+  1Ab = incAb 1g
+
+  invAb : Abelianization → Abelianization
+  invAb =
+    rec
+      squash
+      (λ x → incAb (inv x))
+      (λ (a , b , c) → incAb (inv (a · (b · c)))              ≡⟨ cong incAb (invDistr a (b · c)) ⟩
+                       incAb (inv (b · c) · inv a)            ≡⟨ cong (λ x → incAb (x · inv a)) (invDistr b c) ⟩
+                       incAb ((inv c · inv b) · inv a)        ≡⟨ cong incAb (sym (lid ((inv c · inv b) · inv a))) ⟩
+                       incAb (1g · ((inv c · inv b) · inv a)) ≡⟨ comm 1g (inv c · inv b) (inv a) ⟩
+                       incAb (1g · (inv a · (inv c · inv b))) ≡⟨ cong incAb (lid (inv a · (inv c · inv b))) ⟩
+                       incAb (inv a · (inv c · inv b))        ≡⟨ comm (inv a) (inv c) (inv b) ⟩
+                       incAb (inv a · (inv b · inv c))        ≡⟨ cong incAb (sym (lid (inv a · (inv b · inv c)))) ⟩
+                       incAb (1g · (inv a · (inv b · inv c))) ≡⟨ comm 1g (inv a) (inv b · inv c) ⟩
+                       incAb (1g · ((inv b · inv c) · inv a)) ≡⟨ cong incAb (lid ((inv b · inv c) · inv a)) ⟩
+                       incAb ((inv b · inv c) · inv a)        ≡⟨ cong (λ x → incAb (x · inv a)) (sym (invDistr c b)) ⟩
+                       incAb (inv (c · b) · inv a)            ≡⟨ cong incAb (sym (invDistr a (c · b))) ⟩
+                       incAb (inv (a · (c · b))) ∎)
+
+  assocAb : (x y z : Abelianization) → x ·Ab (y ·Ab z) ≡ (x ·Ab y) ·Ab z
+  assocAb =
+    elimProp3
+      (λ x y z → squash (x ·Ab (y ·Ab z))
+      ((x ·Ab y) ·Ab z))
+      (λ x y z → cong incAb (assoc x y z))
+
+  ridAb : (x : Abelianization) → x ·Ab 1Ab ≡ x
```

Comment:
should be named `·AbIdR` according to the [new guide lines](https://github.com/agda/cubical/blob/master/Cubical/Algebra/NAMING.md) ?

Reply:
same here